app/lib/bk/compat/parsers/bitbucket/caches.rb

```diff
@@ -1,8 +1,5 @@
 # frozen_string_literal: true
 
-require_relative '../../pipeline/step'
-require_relative '../../pipeline/plugin'
-
```

Comment:
This file still uses some classes so it should include the dependencies explicitly:

```suggestion
require_relative '../../models/plugin'
require_relative '../../models/steps/command'

```

Reply:
Added back in 👍